manifests/09-prometheus-rules.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: image-registry-operator-alerts
+  name: image-registry-operator-alerts
+  namespace: openshift-image-registry
+spec:
+  groups:
+  - name: ImageRegistryOperator
+    rules:
+    - alert: ImageRegistryMissingStorageConfiguration
+      expr: max_over_time(image_registry_operator_storage_configured[48h]) == 0
```

Comment:
This will only fire 48h after a config is missing, is that intentional?

Reply:
Yes, that is the idea. As to configure a storage for image registry may be a "day 2" operation the plan is to give the administrator two days before starting to complain that there is no storage configured.